src/content/contribute/plugin-patterns.md

```diff
@@ -78,6 +78,8 @@ module.exports = MyPlugin;
 
 You may also feed new file paths into the watch graph to receive compilation triggers when those files change. Simply push valid file paths into the `compilation.fileDependencies` array to add them to the watch. Note: the `fileDependencies` array is rebuilt in each compilation, so your plugin must push its own watched dependencies into each compilation to keep them under watch.
 
+W> Note that `compilation.fileDependencies`, `compilation.contextDependencies` and `compilation.missingDependencies` are now a `Set` instead of a `Sortable Set` and thus no longer sorted.
```

Comment:
Can we get rid of the Note as this is already in a highlight (stylistically)
```suggestion
W> Since webpack 4, `compilation.fileDependencies`, `compilation.contextDependencies` and `compilation.missingDependencies` are now a `Set` instead of a `Sortable Set` and thus no longer sorted.
```

I am not sure if its since webpack 4, from which changelog did you catch this?

Reply:
The issue was labelled webpack 5

Reply:
All right, merging when CI passes